src/test/java/core/service/CreateReportToCsvTest.java

```diff
@@ -0,0 +1,82 @@
+package core.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.dao.CsvFileReaderService;
+import core.dao.DataToString;
+import core.dao.DataToStringImpl;
+import core.dao.FileReaderService;
+import core.db.Storage;
+import core.model.Operations;
+import core.strategy.AmountHandler;
+import core.strategy.BalanceAmountHandler;
+import core.strategy.PurchaseAmountHandler;
+import core.strategy.ReturnAmountHandler;
+import core.strategy.SupplyAmountHandler;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CreateReportToCsvTest {
+    private static DataToString dataToString;
+    private static FileReaderService fileReaderService;
+    private static CreateReportToFile createReportToFile;
+    private static final String STRING_FORMAT_FOR_WRONG =
+            "Wrong operation! expected: %s But was: %s";
+    @BeforeAll
+    static void beforeAll() {
+        Map<Operations, AmountHandler> activitiesHandlerMap = new HashMap<>();
+        activitiesHandlerMap.put(Operations.BALANCE, new BalanceAmountHandler());
+        activitiesHandlerMap.put(Operations.SUPPLY, new SupplyAmountHandler());
+        activitiesHandlerMap.put(Operations.RETURN, new ReturnAmountHandler());
+        activitiesHandlerMap.put(Operations.PURCHASE, new PurchaseAmountHandler());
+        ActivitiesStrategy activitiesStrategy = new ActivitiesStrategyImpl(activitiesHandlerMap);
+        fileReaderService = new CsvFileReaderService(activitiesStrategy);
+        dataToString = new DataToStringImpl();
+        createReportToFile = new CreateReportToCsv();
+    }
+
+    @Test
+    void testForCorrectData() {
+        fileReaderService.getData("src/test/resources/FruitShopChornovola.csv");
+        createReportToFile.write(dataToString.generateString(),
```

Comment:
fileReaderService.getData();
        dataToString.generateString(),

these are separate services that should be tested in separate tests
here we can just write some hardcoded string to the file and then check if it was written